hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSNamesystem.java

```diff
@@ -1594,10 +1595,14 @@ private void setMetaEnabledInt(final String src,
       EntityManager.update(dirNode);
     }
   
-    if(metaEnabled) {
+    if(isMetaEnabled(metaStatus)) {
       logMetadataEvents(fileTree);
     }
   }
+  
+  private boolean isMetaEnabled(MetaStatus metaStatus) {
```

Comment:
Why not call metaStatus.isMetaEnabled() directly?

Reply:
Old code - didn't fix it everywhere when i added the method to the MetaStatus.
Fixed now.